Undefining __OBJC__ prevents Metal headers from being included.
Supertuxkart will fall back to OpenGL at run-time.

--- lib/irrlicht/source/Irrlicht/MoltenVK.mm.orig
+++ lib/irrlicht/source/Irrlicht/MoltenVK.mm
@@ -1,6 +1,8 @@
 #include "MoltenVK.h"
 #include "SDL_vulkan.h"
+#undef __OBJC__
 #include "vulkan_wrapper.h"
+#define __OBJC__
 
 #include <dlfcn.h>
 #import <AppKit/NSApplication.h>
